src/core/Track.cpp

```diff
@@ -2745,7 +2748,7 @@ void TrackView::mousePressEvent( QMouseEvent * me )
 		{
 			m_action = MoveTrack;
```

Comment:
To here.

Reply:
Tested. Fixes the issue as expected.

Reply:
Direct commit or PR?

Reply:
Direct commit looks fine.

Reply:
Done via https://github.com/LMMS/lmms/commit/e8debf9015979c5d51e5440a78b4dbb2ea2e91b9. 👍